Author: Francois Bissey <francois.bissey@canterbury.ac.nz>
Date:   Tue Feb 11 11:05:00 2015 +1300

	arb seems to be developed and checked against git tree of flint rather than
	a proper release which means some stuff may not be in sync. Case in point
	this function that has changed name between flint 2.4.4 (last release)
	and the git tree at https://github.com/fredrik-johansson/arb

--- acb_modular/test/t-epsilon_arg.c.orig	2015-02-11 10:49:45.092184755 +1300
+++ acb_modular/test/t-epsilon_arg.c	2015-02-11 10:55:28.282719109 +1300
@@ -49,7 +49,7 @@
         fmpz_mul_ui(fmpq_denref(arg), c, 12);
         fmpq_canonicalise(arg);
 
-        fmpq_dedekind_sum(t, d, c);
+        arith_dedekind_sum(t, d, c);
         fmpq_sub(arg, arg, t);
 
         fmpq_clear(t);
